.travis.yml

```diff
@@ -39,7 +39,7 @@ env:
     # END OF TRAVIS MATRIX
 
 before_install:
-  - sudo apt-get update ; sudo apt-get install -y curl apt-transport-https git
+  - sudo apt-get update ; sudo apt-get install -y curl apt-transport-https git netcat net-tools libpq-dev
```

Comment:
Not sure we need to install these, the truth is there's no harm done, but I don't think these are required. I think it's fine if we diverge on the packages installed.